tests/general/KeywordBearTest.py

```diff
@@ -214,3 +214,20 @@ def test_keyword_regex(self):
             self.assertEqual(result[0].message, 'The line contains the keyword'
                                                 " 'Issue #123' which resulted "
                                                 'in a match with given regex.')
+
+    def test_wrong_language(self):
+        section = Section('')
+        section.append(Setting('language', 'anything'))
+        section.append(Setting('keywords', 'TODO'))
+        uut = KeywordBear(section, Queue())
+        dep_results = {
+            'AnnotationBear': HiddenResult(
+                'AnnotationBear',
+                'coalang specification for anything not found.')
+        }
+
+        text = ['a==b']
```

Comment:
put a todo in here and make asserts that the bear still works even if the language is unknown, we just want the warning right?